app.js

```diff
@@ -8,16 +8,20 @@ const fs = require('fs');
 const https = require('https');
 const moment = require('moment');
 const git = require('simple-git/promise');
-const SwaggerParser = require('swagger-parser');
 
 const { errorBuilder, errorHandler } = appRoot.require('errors/errors');
 const { authentication } = appRoot.require('middlewares/authentication');
-const { bodyParserErrorMiddleware } = appRoot.require('middlewares/body-parser-error');
+const { bodyParserError } = appRoot.require('middlewares/body-parser-error');
 const { logger } = appRoot.require('middlewares/logger');
 const { runtimeErrors } = appRoot.require('middlewares/runtime-errors');
+const { validateFilePath } = appRoot.require('utils/fs-operations');
+const { openapi } = appRoot.require('utils/load-openapi');
 
 const serverConfig = config.get('server');
 
+const dbPath = 'tests/unit/mock-data.json';
+validateFilePath(dbPath);
```

Comment:
From our discussion, we should wrap this function into a more generic function so that we can also validate other datasource easily as well.